dadguide/models/monster_stats.py

```diff
@@ -41,24 +41,45 @@ class MonsterStats:
     PLUS_DICT = {'hp': 10, 'atk': 5, 'rcv': 3}
     LV_120_MULT_DICT = {'hp': 10, 'atk': 5, 'rcv': 5}
 
-    def stat(self, monster_model, key: StatType, lv, plus=99, inherit=False, is_plus_297=True,
-             stat_latents: MonsterStatModifierInput = None):
+    def stat(self, monster_model, key: StatType, lv, inherit_monster_model=None, plus=99, inherit=False, is_plus_297=True,
```

Comment:
can you rename `inherit_monster_model` to `inherited_monster` and give it a type checking here instead

also, nitpick: make inherited_monster the last param (not super important but this is just like a less-direct thing I guess)

also, more importantly, put a `*` after `lv` - this will enforce using named params so that if we change signature order (inserting params or reordering or whatever) in the future we aren't making breaking changes